pandas-stubs/io/formats/style.pyi

```diff
@@ -218,6 +218,13 @@ class Styler(StylerRenderer):
         level: Level | list[Level] | None = ...,
         **kwargs: Any,
     ) -> Styler: ...
+    def map_index(
+        self,
+        func: Callable[[Series], npt.NDArray[np.str_] | list[str] | Series[str]],
```

Comment:
Based on the docs, I think this should be `Callable[[Scalar], str]`

Reply:
Thanks, I updated to `Callable[[Scalar], str | None]` to match the docs